webserver/views/test/test_datasets.py

```diff
@@ -201,3 +202,15 @@ def test_recording_info(self):
         self.assert200(resp)
         resp = self.client.get(url_for("datasets.recording_info", mbid=self.test_uuid))
         self.assert404(resp)
+
+    def test_evaluate_location_options(self):
+        self.temporary_login(self.test_user_id)
+        dataset_id = dataset.create_from_dict(self.test_data, author_id=self.test_user_id)
+        resp = self.client.get(url_for("datasets.evaluate", dataset_id=dataset_id))
+        self.assertStatus(resp, 200)
+        evaluate_form = forms.DatasetEvaluationForm()
+        evaluate_form.filter_type.data = forms.DATASET_EVAL_NO_FILTER
+        evaluate_form.evaluation_location.data = forms.DATASET_EVAL_REMOTE
+        evaluate_form.normalize.data = True        
+        resp = self.client.get(url_for("datasets.evaluate", dataset_id=dataset_id, form=evaluate_form))
```

Comment:
You should be clearer about what this test is doing. Are you checking that the form shows when you load the page, or that the form values are set when you submit the page?
If it's the second, this should be a `post` request, not `get`. You should also check that the methods in the views are actually called (https://github.com/metabrainz/acousticbrainz-server/blob/33eb2403678e6126f36c09eb609922b390a429ac/webserver/views/datasets.py#L112)
One way of doing this is by using the mocks module: https://github.com/metabrainz/acousticbrainz-server/blob/e2da6e7c664d061cb6ba4beb504dd60b5eb67db2/db/test/test_data.py
This means that we don't actually have to run the methods